src/main/steps/c100-rebuild/miam/general-reasons/template.njk

```diff
@@ -3,11 +3,8 @@
 
 {% block page_content %}
 
-<span class="govuk-caption-l">{{section}}</span>
-<h1 class="govuk-heading-xl">{{title}}</h1>
-
+<span class="govuk-heading-l">{{section}}</span>
```

Comment:
Pls follow standards for captions and title in terms of markup & styles

Reply:
But this is the expected font from the figma for the title

Reply:
Please refer to other template and follow the standard markup of HTML contents

Reply:
Use the below markup.

  <h1 class="govuk-heading-l">
    <span class="govuk-caption-l">{{caption}}</span>
    {{title}}
  </h1>

Reply:
```
  <h1 class="govuk-heading-l">
    <span class="govuk-caption-l">{{caption}}</span>
    {{title}}
  </h1>
<div class="govuk-body"> {{content}} </div>
```